index.bs

```diff
@@ -1632,12 +1658,13 @@ The <dfn method for=URL id="dfn-revokeObjectURL">revokeObjectURL(|url|)</dfn> st
 
 1. Let |url record| be the result of [=URL parser|parsing=] |url|.
```

Comment:
Maybe change this to urlRecord while here? Having these as separate words is confusing with the url variable.